vimeo/videos.go

```diff
@@ -226,7 +226,8 @@ func (v Video) GetID() int {
 // UploadVideoRequest specifies the optional parameters to the
 // uploadVideo method.
 type UploadVideoRequest struct {
-	FileName string  `json:"file_name"`
```

Comment:
`file_name` required field for [replace video](https://developer.vimeo.com/api/upload/videos)